Sources/SwiftGraphQL/HTTP+WebSockets.swift

```diff
@@ -194,10 +243,14 @@ public struct GraphQLSocketMessage: Codable {
     public enum MessageType: String, Codable {
         case connection_init
         case connection_ack
-        case subscribe
+        case start
         case next
         case error
         case complete
+        case ka
+        case connection_error
+        case connection_terminate
+        case data
```

Comment:
You changed the protocol. I can't find it now but we had a discussion originally which protocol to support and we decided the `graphql-transport-ws` https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md 

You still set that protocol in the header so the server will reply with the original message types.